### PR TITLE
feat: add wordpress template

### PR DESCRIPTION
**Pull request by balanceee**

### CODE REVIEW

**Review by robot9706 — APPROVED** [review]

File: web/crux-ui/e2e/template.spec.ts [hunk 1]
```diff
@@ -54,3 +54,11 @@ test('creating a simple product from a template should work (Strapi)', async ({
 test('creating a complex product from a template should work (Strapi)', async ({ page }) => {
   await testComplexTemplate(page, 'Strapi', 'Strapi-complex', 2)
 })
+
+test('creating a simple product from a template should work (WordPress)', async ({ page }) => {
+  await testSimpleTemplate(page, 'WordPress', 'WordPress-simple', 2)
+})
+
+test('creating a complex product from a template should work (WordPress)', async ({ page }) => {
```
Comment (polaroi8d) [inline web/crux-ui/e2e/template.spec.ts:62]: 🤯

File: web/crux/templates/wordpress.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            "key": "MYSQL_DATABASE",
+            "value": "wordpress"
+          },
+          {
+            "key": "MYSQL_USER",
+            "value": "wpuser"
+          },
+          {
+            "key": "MYSQL_RANDOM_ROOT_PASSWORD",
+            "value": "1"
```
Comment (polaroi8d) [inline web/crux/templates/wordpress.json:38]: Add a random string, please.

Comment (polaroi8d) [reply web/crux/templates/wordpress.json:38]: Nevermind.

**Review by polaroi8d — APPROVED** [review]
LGTM

**Pull request merged**